### DISCUSSION TITLE
您网站的哪些部分使用了 Blazor Server？

### DISCUSSION THREAD

**Discussion by BrandonStudio** [ldqk/Masuit.MyBlogs#21] [Q&A]
我自己做的 Blazor Server 网站以及网络中的其他一些 Blazor Server 网站的前端响应都很慢，也就是延时很高，但我看您的网站就没有这样的问题，是因为您主要使用了MVC框架吗？
我想询问您网站具体哪些组件使用的 Blazor Server 以便我了解 Server Side Blazor 的延时特点

**Comment by ldqk** [comment]
响应很慢很有可能是你写了垃圾代码导致运行慢的

**Reply by BrandonStudio** [reply]
响应缓慢指的是诸如 单击日历类型的输入框要过1秒日历选择框才会出现、处理输入框中的按键按下事件时待按键按下若干毫秒后事件才会被响应 等

**Reply by BrandonStudio** [reply]
同样的代码在 Blazor WebAssembly 中就没有这样的问题。

**Reply by ldqk** [reply]
本地运行的时候慢吗？本地运行都慢的话说明是代码写的有问题

**Reply by BrandonStudio** [reply]
本地运行不慢

**Reply by ldqk** [reply]
那很有可能就是你的服务器网络卡了
